src/main/java/usi/si/seart/model/MetricLanguage.java

```diff
@@ -0,0 +1,48 @@
+package usi.si.seart.model;
+
+import lombok.*;
+import lombok.experimental.FieldDefaults;
+import org.hibernate.annotations.Fetch;
+import org.hibernate.annotations.FetchMode;
+
+import javax.persistence.*;
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
```

Comment:
Do not use wildcard imports, import everything explicitly. This should be configurable somewhere in the IDE settings